specutils/spectra/spectrum_mixin.py

```diff
@@ -65,6 +65,7 @@ def spectral_axis(self):
         return self._spectral_axis
 
     @property
+    @deprecated('v1.0', alternative="spectral_axis.unit")
```

Comment:
Shouldn't this be either:
```suggestion
    @deprecated('v1.1', alternative="spectral_axis.unit")
```
or
```suggestion
    @deprecated('v1.0.1', alternative="spectral_axis.unit")
```

Since 1.0 is already out without the deprecation in there.